partner_compassion/models/mail_activity.py

```diff
@@ -0,0 +1,20 @@
+##############################################################################
+#
+#    Copyright (C) 2020 Compassion CH (http://www.compassion.ch)
+#    @author: Théo Nikles <theo.nikles@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields
+
+
+class MailActivity(models.Model):
+    _inherit = "mail.activity"
+
+    # Override domain to consider only internal users
+    user_id = fields.Many2one(
+        "res.users", "Assigned to",
+        default=lambda self: self.env.user,
+        index=True, required=True,
```

Comment:
When overriding you can only replace the attributes you want and don't specify the others that were already defined in the parent.
```suggestion
```